false/AUTOTEST1/AAT/AuthorisationCaseEvent.json

```diff
@@ -0,0 +1,43 @@
+[ {
```

Comment:
what are these `false` and `true` sub directories used for ?

Reply:
pushed by mistake, while meant to be ignored from source control. removed.